tests/test_translate.py

```diff
@@ -18,6 +18,9 @@
 
 OUTDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "output")
 GPU_MAX_ERR = 1e-10
+GPU_NEAR_ZERO = 1e-15
+
+_near_zero = 1e-18
```

Comment:
I think I'd be ok with changing the near0 for all of them to 1e-15. 1e-18 was somewhat arbitrary (was based on the differences I was seeing with the numpy backend), while we can reason about 1e-15 being at the floating point limit. But, it also makes sense we'd have different thresholds for GPU, so adding this capability is good.